NOTICE

```diff
@@ -11,8 +11,7 @@ License file: nbmolviz/static/3dmol/LICENSE
     D3.js
```

Comment:
Talked to Aaron and the conclusion is to:

  [ ] Keep both the "source file" and "description" bullet points, but point to the file that will be included in node_modules when the project is built.